spec/weather.spec.js

```diff
@@ -0,0 +1,56 @@
+import Weather from '../src/weather.js';
+import { assertEquals } from '../test_framework.js';
+
+console.log('------------------------------');
+console.log('Running tests for weather.js');
+let passCount = 0, failCount = 0;
+
+// Test 1
+{
+	console.log('------------------------------');
+	console.log('Test 1. (getWeather() test). Valid weather => "Sunny"')
+
+	let input, expected, actual, result, airport, plane, weather, srandom;
+	
+	//1. Arrange
+	weather = new Weather();
```

Comment:
An alternate considered was to use a seeded randomizer to ensure consistent random behaviour.
- Could prove to be more useful for other challenges.